src/editors/containers/TextEditor/components/ImageSettingsModal/DimensionControls.jsx

```diff
@@ -42,18 +42,14 @@ export const DimensionControls = ({
     <div className="mt-4.5">
       <Form.Control
         className="dimension-input"
-        type="number"
```

Comment:
Did you look into the feasibility of making the accepted pattern integers and the percent symbol and preventing default otherwise? the freedom to might make people add 'px', etc.